lib/cutting_edge/workers/dependency.rb

```diff
@@ -3,14 +3,15 @@
 require File.expand_path('../../versions.rb', __FILE__)
 require File.expand_path('../../langs.rb', __FILE__)
 require File.expand_path('../helpers.rb', __FILE__)
+require 'hashdiff'
```

Comment:
Need to move `hashdiff` from `Gemfile` to `gemspec` if we go with that implementation